sys-boot/refind/refind-0.11.4.ebuild

```diff
@@ -0,0 +1,197 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs flag-o-matic multiprocessing
+
+DESCRIPTION="The UEFI Boot Manager by Rod Smith"
+HOMEPAGE="https://www.rodsbooks.com/refind/"
+
+SRC_URI="mirror://sourceforge/project/${PN}/${PV}/${PN}-src-${PV}.tar.gz"
+
+LICENSE="BSD GPL-2 GPL-3 FDL-1.3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+FS_USE="btrfs +ext2 +ext4 hfs +iso9660 ntfs reiserfs"
+IUSE="${FS_USE} doc custom-cflags"
+
+DEPEND="|| ( >=sys-boot/gnu-efi-3.0.2 >=sys-boot/udk-2018-r1 )"
+
+DOCS="NEWS.txt README.txt docs/refind docs/Styles"
+
+UDK_ROOT="${EPREFIX}"/usr/share/udk
+[[ -d /usr/share/udk ]] && UDK_ROOT=/usr/share/udk
+UDK_WORKSPACE="${WORKDIR}/udk"
+
+pkg_pretend() {
+	if use custom-cflags; then
+		ewarn
+		ewarn "You have enabled building with USE=custom-cflags. Be aware that"
+		ewarn "using this can result in EFI binaries that fail to run and may"
+		ewarn "fail to build at all. This is strongly advised against by upstream."
+		ewarn
+		ewarn "See https://bugs.gentoo.org/598587#c3 for more information"
+		ewarn
+	fi
+}
+
+pkg_setup() {
+	if use x86; then
+		export EFIARCH=ia32
+		export BUILDARCH=ia32
+	elif use amd64; then
+		export EFIARCH=x64
+		export BUILDARCH=x86_64
+	else
+		# Try to support anyway
+		export BUILDARCH=$( uname -m | sed s,i[3456789]86,ia32, )
+		if [[ ${BUILDARCH} == "x86_64" ]] ; then
+			export EFIARCH=x64
+		else
+			export EFIARCH=${ARCH}
+		fi
+	fi
+}
+
+src_prepare() {
+	default
+
+	# bug 598647 - PIE not supported
+	sed -e 's:CFLAGS          =:& -fno-PIE:' -i Make.common || die
```

Comment:
Don't match exact whitespace, it's fragile. Can't you just e.g. `/^CFLAGS/s:$: -fno_PIE:`?

Reply:
I'd never think of this notation. Now I want to review all my `sed`s! ;)